src/templates/artworks.js

```diff
@@ -80,6 +80,7 @@ query ArtworksQuery($id: String!) {
   }
    markdownRemark(id: { eq: $id }) {
      frontmatter {
+       id
```

Comment:
i forgot to re-add this property in the query and that was the reason of the error, in Layout it was impossible to gett the necessary id to swich between different lang. pages
https://github.com/kalwalt/gatsby-starter-i18n-bulma/blob/dd1efc6e2d54b36f1fd5e9bcc3916b7633479bdb/src/components/Layout.js#L95